### PR TITLE
Set bodyLimit size to 15MB

### PR DESCRIPTION
**Pull request by AntonW**
We need an bigger fastify bodyLimit for documents larger than 1MiB. We found documents larger than 7MiB.
This value is harcoded because the only possible configuration-options are via the config-package.
The fastify-env-plugin seems to be a better solution, but this requires an upgrade to fastify 4.X.

### CODE REVIEW

File: backend/server.js [hunk 1]
```diff
@@ -4,6 +4,7 @@ import config from 'config'
 
 const app = Fastify({
   logger: true,
+    bodyLimit: 10048576
```
Comment (tschmidtb51) [inline backend/server.js:7]: @AntonW I would prefer not to have that hard-coded here. Especially, not the "random" value of 10 MiB. The standard [suggests to use 15 MiB](https://docs.oasis-open.org/csaf/csaf/v2.0/os/csaf-v2.0-os.html#c1-file-size).

**Review by tschmidtb51 — CHANGES_REQUESTED** [review]
I'm sorry but I couldn't follow your explanation about the hardcoded-value. If it is possible via the config package, why don't you use that option? AFAIR we already set some values from a config file...

**Comment by tschmidtb51** [comment]
@domachine: Any additional comments?

**Comment by AntonW** [comment]
You can now configure the bodylimit using the configuration-files. As I wrote earlier: it would be better to use fastify-env. But this needs an upgrade to fastify 4.X

**Comment by domachine** [comment]
> @domachine: Any additional comments?

No, and I don't understand why `fastify-env` should be superior to `config`, since it can map environment variables as well.

File: backend/server.js (hunk 1 above)
Comment (tschmidtb51) [reply backend/server.js]: @AntonW Thanks for changing it.

File: backend/config/default.json [hunk 2]
```diff
@@ -1,4 +1,5 @@
 {
   "ip": "127.0.0.1",
-  "port": 8082
+  "port": 8082,
+  "bodyLimit": 15000000
```
Comment (tschmidtb51) [inline backend/config/default.json:4]: As discussed today:
```suggestion
  "bodyLimit": 15728640

**Review by tschmidtb51 — COMMENTED** [review]
One minor change, before ready to accept.

File: backend/config/default.json (hunk 2 above)
Comment (tschmidtb51) [reply backend/config/default.json:4]: As discussed in the CSAF Editor meeting this should be 15728640 bytes.

File: backend/config/development.json [hunk 3]
```diff
@@ -1,4 +1,5 @@
 {
   "ip": "0.0.0.0",
-  "port": 8082
+  "port": 8082,
+  "bodyLimit": 15000000
```
[the remark this answers is not in the document]
Comment (tschmidtb51) [reply backend/config/development.json:4]: As discussed in the CSAF Editor meeting this should be 15728640 bytes.

**Review by tschmidtb51 — APPROVED** [review]
LGTM.

**Review by sthagen — APPROVED** [review]
From the sidelines (as co-editor of CSAF v2.0) please use the multiples of 2 measure `15 * (2**20)` as the target of the limit is to avoid storage / memory related problems.

**Pull request merged**